src/features/rates.api.ts

```diff
@@ -0,0 +1,15 @@
+import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react'
+
+export const ratesApi = createApi({
```

Comment:
We use fakeBaseQuery and queryFn when calling gPRC. But for simple http you can use that https://redux-toolkit.js.org/rtk-query/api/fetchBaseQuery#using-fetchbasequery

Reply:
Yeah, I just used that as a 'skeleton'/'placeholder' rn. I'm still working on it.